### ISSUE TITLE
Enhance the abstract/description text fields in the Summary page to deal with text formatting

### ISSUE THREAD

**Issue by jonquet** [ontoportal-lirmm/bioportal_web_ui#659]
Some ontologies (e.g. LEXEAU) have a very long text to describe the resource. 
This issues is about enhancing the display of the abstract and description metadata properties to deal with text formatting (new lines, bold, underline etc.)
The challenge is to enable this while keeping the page "coherent" from one ontology to the other.

It means to enable a new text input component in the Edit submisison page, that will be based on a tiny HTML text editor.

**Comment by JeanLouisJanin** [comment]
I agree with that proposal. Would it be possible to include in the description format hypertext links with web pages ?

**Comment by maboukerfa** [comment]
@jonquet still relevant ?
